### PR TITLE
Implementing basic CRUD oprations

### PR DESCRIPTION
**Pull request by SeyedMojtaba1**

### CODE REVIEW

File: Seyed/TypeORM CRUD/index2.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    try{
+        var check = await repo.exists({
+            where:{
+                fName: newUser.fName,
+                lName: newUser.lName
+            }
+        })
+
+        if(!check){
+        await repo.save(newUser);
```
Comment (fatemeh-ra) [inline Seyed/TypeORM CRUD/index2.ts:24]: what about code indents?  
Try using insert operator when you know it's a new row. Search for differences between save and insert queries in typeorm.

File: Seyed/TypeORM CRUD/index2.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        }else{
+            console.log("The desired user does not exist!")
+        }
+
+    }catch(error){
+        console.error(error)
+    }
+}
+
+export default dataSource
```
Comment (fatemeh-ra) [inline Seyed/TypeORM CRUD/index2.ts:108]: why??

File: Seyed/TypeORM CRUD/index2.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+async function deleteUser(userId: number, repo: Repository<User>){
+    try{
+        const check = await repo.exists({
+            where:{
+                id: userId
+            }
+        })
+        
+        if(check){
+            const user = await repo.findOne({
```
Comment (fatemeh-ra) [inline Seyed/TypeORM CRUD/index2.ts:77]: why did you find the user? Too many queries for a simple delete operation.

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

File: Seyed/TypeORM CRUD/index2.ts (hunk 2 above)
Comment (SeyedMojtaba1) [reply Seyed/TypeORM CRUD/index2.ts:108]: I fixed it.

File: Seyed/TypeORM CRUD/index2.ts (hunk 3 above)
Comment (SeyedMojtaba1) [reply Seyed/TypeORM CRUD/index2.ts:77]: I fixed it.

**Review by fatemeh-ra — APPROVED** [review]

**Pull request merged**